# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1551131467 0
#      Mon Feb 25 21:51:07 2019 +0000
# Node ID 7e48abcc10322899aceed17158999748cebd86a1
# Parent  45fc6505a803607888bc3130fae5e143c01292fb
* test/bug-45351/module.mk: Fix cut & paste error in previous change.

diff --git a/test/bug-45351/module.mk b/test/bug-45351/module.mk
--- a/test/bug-45351/module.mk
+++ b/test/bug-45351/module.mk
@@ -1,5 +1,5 @@
-bug_54995_TEST_FILES = \
+bug_45351_TEST_FILES = \
   %reldir%/bug45351.m \
   %reldir%/bug-45351.tst
 
 TEST_FILES += $(bug_45351_TEST_FILES)
